manim/cli/project/templates/MovingCamera.py

```diff
@@ -0,0 +1,8 @@
+class MovingCamera(MovingCameraScene):
```

Comment:
[Same as above](https://github.com/ManimCommunity/manim/blob/master/manim/camera/moving_camera.py#L28)

Reply:
Okay i will change it to MovingCameraTemplate and Graph to GraphTemplate